nengo/spa/vocab.py

```diff
@@ -81,6 +81,10 @@ def __init__(self, dimensions, randomize=True, unitary=False,
         self.include_pairs = include_pairs
         self._identity = None
         self.rng = rng
+        self.read_only = False
+
+        # The parent of a non-subset vocabulary is itself
+        self.parent = self
```

Comment:
Oh. Hah. This is so that I can shortcut the comparison in the transform_to function. XD